training/model_management/components/import_model/spec.yaml

```diff
@@ -104,8 +119,10 @@ outputs:
 
 jobs:
   download_model:
-    component: azureml:download_model:0.0.3
+    component: azureml://registries/azureml/components/download_model/versions/0.0.3
```

Comment:
Why expand these? The current approach (`azureml:download_model:0.0.3`) attempts to resolve the component in the registry where the pipeline component is being created.